### PR TITLE
Mark `cc_static_library` as experimental in docs

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

**Comment by fmeum** [comment]
@bazel-io fork 7.4.0

**Comment by iancha1992** [comment]
Friendly ping @pzembrod

File: src/main/starlark/builtins_bzl/common/cc/experimental_cc_static_library.bzl [hunk 1]
```diff
@@ -244,6 +244,9 @@ def _cc_static_library_impl(ctx):
 cc_static_library = rule(
     implementation = _cc_static_library_impl,
     doc = """
+<b>This rule is currently experimental and can only be used with the <code>
+--experimental_cc_static_library</code> flag.
```
Comment (fmeum) [inline src/main/starlark/builtins_bzl/common/cc/experimental_cc_static_library.bzl:248]:
```suggestion
--experimental_cc_static_library</code> flag.</b>
```

**Review by pzembrod — APPROVED** [review]

File: src/main/starlark/builtins_bzl/common/cc/experimental_cc_static_library.bzl [hunk 2]
```diff
@@ -244,6 +244,9 @@ def _cc_static_library_impl(ctx):
 cc_static_library = rule(
     implementation = _cc_static_library_impl,
     doc = """
+<b>This rule is currently experimental and can only be used with the <code>
```
[the remark this answers is not in the document]
Comment (fmeum) [reply src/main/starlark/builtins_bzl/common/cc/experimental_cc_static_library.bzl:247]: Bold yes, but it should be closed.

**Comment by iancha1992** [comment]
The changes in this PR have been included in [Bazel 7.4.0 RC1](https://github.com/bazelbuild/bazel/releases/tag/7.4.0rc1). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting `USE_BAZEL_VERSION=7.4.0rc1`. Thanks!

**Comment by iancha1992** [comment]
@bazel-io fork 8.0.0

**Pull request closed without merging**